hack/test-gce/gce_delete.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+set -e
+
+
+delete_instance_by_number() {
+    # Enter the nodeagent number which you want to delete 
+    echo -e "## Enter nodeagent number which you want to delete in your test:"
+    read instance_num
+    echo -e "\n"
+
+    # delete virtual machine instance
+    gcloud compute instances delete davidzhu-fornaxcore --zone=us-central1-a
+
+    # instance_num=2
+    for ((i = 1; i<=$instance_num; i++))
+    do
+        instance_name='davidzhu-nodeagent-'$i
```

Comment:
same as above

Reply:
I will use async to do it in deployed sh file